.neutrinorc.js

```diff
@@ -1,65 +1,81 @@
-            'no-console': 'off',
-            // Allowing shadowing variable that share the same context as the outer scope
-            'no-shadow': 'off'
-          },
-          overrides: [
-            {
-              files: ['packages/create-project/commands/init/templates/**'],
-              rules: {
-                // The dependencies in create-project's templates are installed by
-                // by create-project and so are expected to be missing from package.json.
-                'import/no-extraneous-dependencies': 'off'
-              }
+    [
+      './packages/airbnb',
+      {
+        // See the package.json `lint` script for which files are linted.
+        // Excludes are managed via `.eslintignore`.
+        eslint: {
+          baseConfig: {
+            extends: ['plugin:prettier/recommended'],
+            env: {
+              browser: true,
+              jest: true,
+              mocha: true,
+              node: true,
             },
-            {
-              files: ['packages/create-project/commands/init/templates/preact/**'],
-              settings: {
-                react: {
-                  pragma: 'h'
-                }
-              },
-              rules: {
-                // With Preact the use of `class` is recommended over `className`,
-                // so we have to add `class` to the ignore list, to prevent:
-                // `Unknown property 'class' found, use 'className' instead`
-                'react/no-unknown-property': ['error', { ignore: ['class'] }]
-              }
+            rules: {
+              'prettier/prettier': [
+                'error',
+                {
+                  singleQuote: true,
+                  jsxBracketSameLine: true,
+                  trailingComma: 'all',
+                  proseWrap: 'always',
+                  endOfLine: 'lf',
+                },
+                { usePrettierrc: false },
+              ],
+              // Allow using console since most of the code in this repo isn't run in a browser.
+              'no-console': 'off',
+              // Allowing shadowing variable that share the same context as the outer scope
+              'no-shadow': 'off',
             },
-            {
-              files: ['packages/*/test/*'],
-              rules: {
-                // The tests need to do non-global require() to test the presets.
-                'global-require': 'off',
-                // This rule doesn't handle devDependencies being defined
-                // in the monorepo root package.json.
-                'import/no-extraneous-dependencies': 'off'
-              }
-            }
-          ]
-        }
-      }
-    }]
-  ]
+            overrides: [
+              {
+                files: [
+                  'packages/create-project/commands/init/templates/**',
+                  'packages/create-project/commands/init/templates/*/.eslintrc.js',
```

Comment:
If `.eslintignore` is changed to use `!.*.js`, then locally I get an `import/no-extraneous-dependencies` error in: 
`packages/create-project/commands/init/templates/mocha/.mocharc.js`. 

Perhaps this entry should also be changed to use a wildcard? eg:
`packages/create-project/commands/init/templates/*/.*.js`